doc/nrf/app_dev/device_guides/nrf70/wifi_advanced_security_modes.rst

```diff
@@ -0,0 +1,78 @@
+.. _ug_nrf70_wifi_advanced_security_modes:
+
+nRF70 Series advanced security modes
+####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Enterprise security
+*******************
+
+The nRF70 Series devices support Wi-Fi enterprise security, which is a more secure form of Wi-Fi security compared to Wi-Fi personal security.
```

Comment:
```suggestion
The nRF70 Series devices support Wi-Fi® enterprise security, which is a more secure form of Wi-Fi security compared to Wi-Fi personal security.
```

Reply:
Thanks, I will take this up in next push or if there is no push (here's hoping) in a subsequent PR.